.chloggen/1413-cache-scrape-config-response.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: "target allocator"
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Save the scrape config response in the HTTP server upon relevant confif change, instead of building it on every handler call. At the same time, this avoids data race when accessing the scrape configs map."
```

Comment:
```suggestion
note: "Save the scrape config response in the HTTP server upon relevant config change, instead of building it on every handler call. At the same time, this avoids data race when accessing the scrape configs map."
```